api/routing/router_test.go

```diff
@@ -164,6 +165,28 @@ var _ = Describe("Router", func() {
 			Expect(res).To(HaveHTTPBody(MatchJSON(`{"not":"allowed"}`)))
 		})
 	})
+
+	When("a request comes with trailing slash", func() {
```

Comment:
I believe that this context is testing whether the trailing slash middleware from chi is working correctly and making sure that this is the case is out of scope for Korifi tests. That should have been tested by chi itself.

From my point of view, having an e2e test that makes sure that requests with trailing slash do not fail should be enough. TL;DR I think you could remove the `When("a request comes with trailing slash"` context

Reply:
Done